encoder/src/main/java/com/pedro/encoder/input/video/Camera1ApiManager.java

```diff
@@ -63,23 +64,26 @@
 
   private FaceDetectorCallback faceDetectorCallback;
 
-  public Camera1ApiManager(SurfaceView surfaceView, GetCameraData getCameraData) {
+  public Camera1ApiManager(SurfaceView surfaceView, GetCameraData getCameraData, CameraSwitchCallback cameraSwitchCallback) {
```

Comment:
OpenGlViews works only for Camera2 api, so there is no need to add this callback here I guess?

Reply:
As discussed, it works for Camera1